lib/src/rules/null_closures.dart

```diff
@@ -92,66 +92,70 @@ class NonNullableFunction {
   InterfaceTypeDefinition _typeDefinition;
 
   NonNullableFunction(this.library, this.type, this.name,
-      {this.positional = const <int>[], this.named = const <String>[]});
+      {this.positional: const <int>[], this.named: const <String>[]});
```

Comment:
These changes are also not related to `new` and `const` and might want to be preserved.